src/other/pages/what-is-new/index.scss

```diff
@@ -7,7 +7,14 @@
 @include accessory-story-content;
 @include accessory-story-wrapper;
 
-.what-is-new__text-with-link {
-    @include text(t2);
-    max-width: 750px;
-}
\ No newline at end of file
+.what-is-new {
+    &__text-with-link {
+        @include text(t2);
+        max-width: 750px;
+    }
+
+    &__response-date {
+        // The important flag is needed, because the component iself sets its styles to the host element internally.
+        margin-bottom: 20px !important;
```

Comment:
could wrap header in a `<div>` and set styles there

Reply:
This would be even worse than an important statement on a BEM class.